Frontend/ui-driver/src/Screens/HomeScreen/Handler.purs

```diff
@@ -41,6 +42,8 @@ import Debug
 
 homeScreen :: FlowBT String HOME_SCREENOUTPUT
 homeScreen = do
+  liftFlowBT $ Events.endMeasuringDuration "Flow.mainFlow"
+  liftFlowBT $ Events.endMeasuringDuration "Flow.initialFlow"  
```

Comment:
start time of main and initial flow must be different na? with this metric will be correct?